editions/tw5.com/tiddlers/widgets/ActionDeleteFieldWidget.tid

```diff
@@ -16,6 +16,7 @@ The ''action-deletefield'' widget is invisible. Any content within it is ignored
 |!Attribute |!Description |
 |$tiddler |The title of the tiddler whose fields are to be modified (if not provided defaults to the [[current tiddler|Current Tiddler]]) |
 |$field |Optional name of a field to delete |
+|$timestamp |Specifies whether the timestamp(s) of the target tiddler will be updated (''modified'' and ''modifier'', plus ''created'' and ''creator'' for newly created tiddlers). Can be "yes" (the default) or "no" |
```

Comment:
this needs `<<.from-version "5.3.4">> Specifies...`

Reply:
Ah, good catch! I've made that change as well